CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,72 @@
+
+# Standards
+The Open Research devroom team is dedicated to providing a harassment-free experience for everyone.
+
+This means that the organisers are commited to facilitating an open and inclusive environment for all participants, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion (or lack thereof), sexual identity and orientation, or technology choices. 
+
+The following behaviors are expected and requested of all devroom participants:
+
+- Be respectful and value each other’s ideas, styles and viewpoints.
+- Be considerate of others' participation, including use of conference time
+- Be inclusive and help new perspectives be heard
+- Be open to learning from others
+- Be mindful of your surroundings and of your fellow participants. Please alert the devroom organisers if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+
+As an overriding general rule, please be intentional in your actions and humble in your mistakes.
+
+We do not tolerate harassment of conference participants in any form – both online and at the in-person event. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties, Twitter, Mastodon and other online media. 
+
+Unacceptable behaviors include the following:
+- Intimidating, harassing, lewd, demeaning, bullying, stalking, trolling or threatening speech or actions
+- Unwelcome sexual attention or physical contact
+- Any real or implied threat of physical harm
+- Sustained disruption of speakers or events (verbally or physically)
+- Publishing others’ private information, such as a physical or email address, without their explicit permission
+- Advocating or encouraging any of the above behaviors
+
+For more information and examples about what constitutes harassment, please refer to OpenCon’s Code of Conduct in Brief and the [Gathering for Open Source Hardware’s](https://openhardware.science/gosh-2017/gosh-code-of-conduct/) examples of behaviour.
+
+Conference participants violating these rules may be asked to leave or removed from the Open Research devroom (virtual or in-person) at the discretion of the organisers. 
+
+# Enforcement 
+Participants asked to stop any harassing behavior are expected to comply immediately.
+
+If a participant engages in harassing behavior, the conference organisers may take any action they deem appropriate, including warning the offender or expulsion from the online or in-person conference room. 
+
+The organising team may take action to redress anything designed to disrupt, or with the clear impact of disrupting, the event or making the environment hostile for any participants.
+
+# Reporting
```

Comment:
We mentioned something here about having two points of contact - @diegantobass and @yochannah? Do we want to have your information here directly, or something less specific in the written form?

Maybe adding more information: "Alternatively, at both the in-person and online events, participants can speak to any of the organisers directly. Two devroom organisers will be designated as points of contact for any participants."